pkgs/by-name/li/librespeed-rust/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "librespeed-rust";
+  version = "1.3.8";
+
+  src = fetchFromGitHub {
+    owner = "librespeed";
+    repo = "speedtest-rust";
+    tag = "v${version}";
+    hash = "sha256-TINIKZefT4ngnEtlMjxO56PrQxW5gyb1+higiSnkE3Q=";
+  };
+
+  postPatch = ''
+    # error: linker `aarch64-linux-gnu-gcc` not found
+    # rm .cargo/config.toml
+    ln -s ${./Cargo.lock} Cargo.lock
+  '';
+
+  cargoLock.lockFile = ./Cargo.lock;
```

Comment:
I might not be entirely up to date here, but I thought that vendoring lockfiles was something we didn't do in nixpkgs anymore? Would it work to just set `cargoHash` instead?

Reply:
The problem is that there isn't an upstream lock file to begin with. I've created an upstream pr which could be fetched with fetchPatch.